app/templates/components/chart-toggle.html

```diff
@@ -0,0 +1,18 @@
+<p class="text-corporate-blue text-center text-h2">
+  {{ toggleText }}
+  <select
+    id="{{ section_id }}_toggle"
+    class="bg-white text-center font-bold border-b-2 border-corporate-blue inline"
```

Comment:
I would not center the label (`text-center`), because it looks a bit more integrated into the "sentence" when it’s left-aligned.

<img width="514" alt="Bildschirmfoto 2023-10-02 um 18 06 37" src="https://github.com/technologiestiftung/innovationserhebung/assets/6429568/54659868-c3bb-47c8-9131-470fdc900b90">

And you can consider adding some hover style (maybe background change and `cursor: pointer`?) to make it more obvious where the "hit zone" is.

Reply:
Jep, good point. I also wasn't sure about the centered Text however That is gonna be Ui's decision. I also added cursor und background